.bbp-project.yaml

```diff
@@ -19,3 +19,9 @@ tools:
       match:
       - ext/.*
       - src/language/templates/*
+  Black:
```

Comment:
Future reference: `ruff` can do linting and black-style format checks.

Reply:
The BBP formatter does: C++, CMake and Python, including installing the correct version of all of those automatically.